Workflows/Environment Synchronization.md

```diff
@@ -0,0 +1,126 @@
+## Environment Synchronization
+A synchronization of environments, or more precisely, a synchronization of the data used in different environments,
+is usually necessary when you need to work in an environment that is as production-like as possible, but you don't have
+access to production itself, for example:
+* when you need to debug a more complex issue occuring in production
+* when you need data that is as close to production as possible (both when it comes to type and amount)
+for development and/or testing purposes
```

Comment:
```suggestion
for development and/or testing purposes.
```

Reply:
I wouldn't add a full stop at the end of a bullet point that is not a full sentence in its own right

Reply:
fudge, the line split weird on my screen, carry on 😄